### DISCUSSION TITLE
请教下，能不能使用unocss

### DISCUSSION THREAD

**Discussion by xiaojieajie** [NervJS/taro#11464] [Q & A]
能不能使用unocss

**Comment by keysking** [comment]
![image](https://user-images.githubusercontent.com/16982187/162576460-6d659e6e-171a-458e-8679-b338c6e57bd9.png)
```typescript
// config/index.js
const UnoCSS = require("@unocss/webpack").default;
const config = {
// ...
  mini: {
      webpackChain: (chain, webpack) => {
        chain.plugin("unocss").use(UnoCSS);
      },
      // ...
  }
// ...
```

**Reply by keysking** [reply]
噢,我没有写完就不小心发布出去了...
是的,我试过了,只需要在webpack里配置unocss的webpack插件即可,
但是有一个问题是,每次我使用了新的css,我都需要重新运行`yarn run dev:weapp`才能正常显示,就好像unocss的webpack只有这时才被触发运行.
我对webpack和taro都不熟悉,所以可能是配置webpack的方式错了, 还希望能有懂的大佬给予纠正,感谢Orz

**Comment by transtone** [comment]
仅供参考： https://github.com/pcdotfan/taro-plugin-tailwind/issues/84

**Comment by MellowCo** [comment]
unocss预设：https://github.com/MellowCo/unocss-preset-weapp/tree/main/examples/taro_webpack4_vue3
